deploy/olm-catalog/manifests/extensions.hive.openshift.io_agentclusterinstalls.yaml

```diff
@@ -140,6 +140,31 @@ spec:
                 required:
                 - name
                 type: object
+              diskEncryption:
+                description: DiskEncryption is the configuration to Enable/disable
+                  disk encryption for cluster nodes.
+                properties:
+                  enableOn:
+                    default: none
+                    description: Enable/disable disk encryption on master nodes, worker
+                      nodes, or all nodes.
+                    enum:
+                    - none
+                    - all
+                    - masters
+                    - workers
+                    type: string
+                  mode:
+                    description: The disk encryption mode to use.
+                    enum:
+                    - tpmv2
+                    - tang
+                    type: string
+                  tangServers:
+                    description: JSON-formatted string containing additional information
+                      regarding tang's configuration
+                    type: string
+                type: object
```

Comment:
same question

Reply:
This is auto-generated by the kubebuilder framework.